bundles/org.openhab.binding.onecta/src/main/java/org/openhab/binding/onecta/internal/handler/OnectaBridgeHandler.java

```diff
@@ -0,0 +1,167 @@
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.onecta.internal.handler;
+
+import static org.openhab.binding.onecta.internal.OnectaBridgeConstants.*;
+
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.onecta.internal.OnectaConfiguration;
+import org.openhab.binding.onecta.internal.api.OnectaConnectionClient;
+import org.openhab.binding.onecta.internal.api.dto.units.Units;
+import org.openhab.binding.onecta.internal.exception.DaikinCommunicationException;
+import org.openhab.binding.onecta.internal.service.DeviceDiscoveryService;
+import org.openhab.core.thing.*;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OnectaBridgeHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Alexander Drent - Initial contribution
+ */
+@NonNullByDefault
+public class OnectaBridgeHandler extends BaseBridgeHandler {
+    private final Logger logger = LoggerFactory.getLogger(OnectaBridgeHandler.class);
+
+    private @Nullable OnectaConfiguration config;
+
+    private @Nullable ScheduledFuture<?> pollingJob;
+
+    private Units units = new Units();
+    private final OnectaConnectionClient onectaConnectionClient = new OnectaConnectionClient();
+
+    public Units getUnits() {
+        return units;
+    }
+
+    private @Nullable DeviceDiscoveryService deviceDiscoveryService;
+
+    /**
+     * Defines a runnable for a discovery
+     */
+    Runnable runnable = new Runnable() {
+        @Override
+        public void run() {
+            if (deviceDiscoveryService != null) {
+                deviceDiscoveryService.startScan();
+            }
+        }
+    };
+
+    public OnectaBridgeHandler(Bridge bridge) {
+        super(bridge);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("initialize.");
+        config = getConfigAs(OnectaConfiguration.class);
+
+        updateStatus(ThingStatus.UNKNOWN);
+        try {
+            onectaConnectionClient.startConnecton(thing.getConfiguration().get(CONFIG_PAR_USERID).toString(),
+                    thing.getConfiguration().get(CONFIG_PAR_PASSWORD).toString());
+
+            if (onectaConnectionClient.isOnline()) {
+                updateStatus(ThingStatus.ONLINE);
+            } else {
+                updateStatus(ThingStatus.OFFLINE);
+            }
+        } catch (DaikinCommunicationException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        }
+
+        pollingJob = scheduler.scheduleWithFixedDelay(this::pollDevices, 10,
```

Comment:
```suggestion
        pollingJob = scheduler.scheduleWithFixedDelay(this::pollDevices, 0
```
Is there a need to wait 10 seconds to execture the first run?